src/app/models/register/register-details.ts

```diff
@@ -0,0 +1,11 @@
+export class RegisterData {
```

Comment:
Try renaming the model interface to User so that the one working on login can also reuse it.

Reply:
Hey @bl4ck4ndbr0wn . Thank you. I have worked on this.